routers/movie.router.js

```diff
@@ -5,8 +5,18 @@ const auth = require('../config/auth').auth;
 var reviewRouter = require('./review.router');
```

Comment:
We need to import lodash first before we can use it:
`const _ = require('lodash');`